server/verify.go

```diff
@@ -125,14 +123,14 @@ func VerifyAttestation(attestation *pb.Attestation, opts VerifyOpts) (*pb.Machin
 }
 
 func pubKeysEqual(k1 crypto.PublicKey, k2 crypto.PublicKey) bool {
-	switch key := k1.(type) {
```

Comment:
this makes sense, but could you link to the exact documentation (I couldn't find it).

Reply:
I see a proposal here that was accepted and closed: https://github.com/golang/go/issues/21704

Reply:
Done, it's in the 1.18 beta docs